sources/Valkey.Glide/ConnectionConfiguration.cs

```diff
@@ -25,9 +25,24 @@ internal record ConnectionConfig
         public Protocol? Protocol;
         public string? ClientName;
         public bool LazyConnect;
+        public bool RefreshTopologyFromInitialNodes;
 
         internal FFI.ConnectionConfig ToFfi() =>
-            new(Addresses, TlsMode, ClusterMode, (uint?)RequestTimeout?.TotalMilliseconds, (uint?)ConnectionTimeout?.TotalMilliseconds, ReadFrom, RetryStrategy, AuthenticationInfo, DatabaseId, Protocol, ClientName, LazyConnect);
+            new(
```

Comment:
Are there doc changes needed for this?
Also, this repo doesn't have a CHANGELOG.md but it probably should since it has had a public preview now.